### PR TITLE
backgroundImage and text on whatwedo header is updated

### PR DESCRIPTION
**Pull request by busratok**
### Description 

- Background image changed and text on whatwedopage header is changed according to new design

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -37,7 +37,7 @@
   resolved "https://registry.npmjs.org/@babel/compat-data/-/compat-data-7.22.9.tgz"
   integrity sha512-5UamI7xkUcJ3i9qVDS+KFDEK8/7oJ55/sJMB1Ge7IEapr7KfdfV/HErR+koZwOfd+SgtFKOKRhRakdg++DcJpQ==
 
-"@babel/core@^7.0.0", "@babel/core@^7.0.0-0", "@babel/core@^7.11.6", "@babel/core@^7.12.3", "@babel/core@^7.8.0":
+"@babel/core@^7.11.6", "@babel/core@^7.12.3":
```
[the remark this answers is not in the document]
Comment (busratok) [reply yarn.lock]: I reverted the changes on yarn.lock

[the remark this answers is not in the document]
Comment (lucaCambi77) [reply yarn.lock]: ok i see there are still `Whakaako` words around, should we also move those to the new name?

[the remark this answers is not in the document]
Comment (lucaCambi77) [reply yarn.lock]: also conflicts here to be fixed

[the remark this answers is not in the document]
Comment (lucaCambi77) [reply yarn.lock]: ok thanks @busratok i see there are very heavy images to load, so I'll create an issue to make lighter images

**Review by lucaCambi77 — APPROVED** [review]

**Pull request merged**